code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -2,7 +2,7 @@
 	SIGNAL_HANDLER_DOES_SLEEP
 	. = COMSIG_KB_ACTIVATED //The return value must be a flag compatible with the signals triggering this.
 
-	if(incapacitated() || lying_angle || istype(loc, /obj/vehicle/multitile/root/cm_armored))
+	if(incapacitated() || lying_angle || isvehicle(loc))
```

Comment:
based on the original code, should this not be `isarmoredvehicle()`?

Reply:
The intent behind these is that if possibly in future we have other non armored vehicles this doesn't need to be updated